custom/icds_reports/sqldata/agg_child_health_monthly.py

```diff
@@ -337,6 +327,38 @@ def get_columns(self, filters):
                 slug='low_birth_weight'
             )
         ]
+        if self.beta:
+            aggregate_columns.insert(13, AggregateColumn(
+                'Percent children between 1-2 years who got immunized with 1st year immunizations',
+                lambda x, y, z: ((x or 0) + (y or 0)) * 100 / float(z or 1),
+                [
+                    SumWhen(
+                        whens=[["age_tranche = :age_24", 'fully_immunized_on_time']],
```

Comment:
should be `age_tranche <=`

Reply:
we want children between 12-24 so <=24 will cover below 1 as well right